start.sh

```diff
@@ -13,7 +13,7 @@ if [ ! -e ${INI_DIR} ]; then
 fi
 
 pushd ${INI_DIR} > /dev/null
-uwsgi rovercode.ini --plugin python
+echo "rovercode service starting"
+./uwsgi rovercode.ini --plugins python,corerouter,http,gevent --gevent 1000 --http-websockets
```

Comment:
You might want to check if uwsgi exists here. Then, if not, you can give a meaningful error message about running setup.sh.